src/applications/vaos/referral-appointments/tests/hooks/useGetProviderByReferral/useGetProviderByReferral.unit.spec.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
```

Comment:
I like the hook. But we should probably update everything that fetches provider to use it as well